system/BaseModel.php

```diff
@@ -721,15 +722,27 @@ public function save($row): bool
     }
 
     /**
-     * This method is called on save to determine if entry have to be updated.
-     * If this method returns false insert operation will be executed
+     * This method is called on save to determine if the entry needs to be updated.
+     * It checks if the ID is not empty and does not already exist in the table.
+     * If this method returns false, an insert operation will be executed.
      *
-     * @param array|object $row Row data
+     * @param array|object $row Data
```

Comment:
Why do change this?

Reply:
> Why do you change this comment?

ah sorry I missed that part, it shouldn't be there